.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,19 @@
+name: Release Drafter
```

Comment:
Is the idea behind the release drafter that we draft a release and then deploy with the tag being the released version? 
Would it be a better idea to hold this off until we CI/CD this out?

Reply:
I think it i good to have it here and does not hurt anything. keeps us in line code artifact and lowers confusion